pkg/ee/metering/deployment.go

```diff
@@ -48,6 +48,8 @@ func deploymentCreator(seed *kubermaticv1.Seed) reconciling.NamedDeploymentCreat
 	return func() (string, reconciling.DeploymentCreator) {
 		//TODO: Add custom values for the metering deployment fields such as seed, interval and output-rotation.
 		return meteringToolName, func(d *appsv1.Deployment) (*appsv1.Deployment, error) {
+			d.Name = meteringToolName
+			d.Namespace = resources.KubermaticNamespace
```

Comment:
This should probably be dynamic to the seed namespace name. Maybe it would be better to use `d.Namespace = seed.Namespace` instead like in [pvc.go](https://github.com/kubermatic/kubermatic/blob/master/pkg/ee/metering/pvc.go#L49)?

Reply:
I think it makes sense to use the namespace provided by the operator. It is the same that is used to deploy the Seed-Controller-Manager.